sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -113,25 +126,32 @@ def check_ortho(M, err_msg):
         [[-0.61330704, -0.00443647,  0.78983213],
          [-0.74697144, -0.32172099, -0.58183269],
          [-0.25668686,  0.94682413, -0.19399983]])
-    assert_array_almost_equal(pls_2.x_weights_, x_weights)
+    x_weights_sign_flip = pls_2.x_weights_ / x_weights
```

Comment:
I think in addition to anything else, comparing to R is good ;)
I'm fine with leaving as is.